### ISSUE TITLE
Expected, Actual message for shouldNotContain

### ISSUE THREAD

**Issue by goreRatzete** [MarkusAmshove/Kluent#20]
I have been using the Collection.`shouldNotContain` method to verify that my Collection does NOT contain an item.

However I found the output on fail quite confusing:

![image](https://cloud.githubusercontent.com/assets/9032394/22931304/6b6f0a80-f2c4-11e6-9740-ae9ea2869be8.png)

It shows the item that I do not want to be in the Collection as Expected, and the Collection as actual.

In other words it looks as if it was actually expected that the item was in the Collection. 

I will probably look into this soon.

Also let's make some progress on common assertions like String.shouldNotBeNullOrBlank !

**Comment by MarkusAmshove** [comment]
Ah yeah, you're right. 

I also found some unfitting messages and changed them in past releases. I wanted to check all messages then, but forgot about it :-)

I'll open issues about the common assertions so we can start on them 


Am 14. Februar 2017 14:51:01 MEZ schrieb Andreas Volkmann <notifications@github.com>:

**Comment by MarkusAmshove** [comment]
I think I'd like to change

```kt
infix fun <T> Iterable<T>.`should not contain`(theThing: T) = if (!this.contains(theThing)) Unit else fail("$this should not contain $theThing", "$theThing", join(this))
```

to 
```kt
infix fun <T> Iterable<T>.`should not contain`(theThing: T) = if (!this.contains(theThing)) Unit else fail("$this should not contain $theThing", "the collection to not contain [$theThing]", join(this))
```

or something like this, since Eclipse and IntelliJ open a diff window for Strings and you can have a nice look at the joined collection.
Any thoughts?

**Comment by MarkusAmshove** [comment]
I did a bit of testing and came up with 2 possible solutions:

![unbenannt](https://cloud.githubusercontent.com/assets/2401875/23059785/2dc9446e-f4fb-11e6-904e-d5d364df88f1.PNG)

![unbenannt2](https://cloud.githubusercontent.com/assets/2401875/23059786/2dca3112-f4fb-11e6-8053-45b5ac668c4b.PNG)


What do you think?

**Comment by goreRatzete** [comment]
Well I do believe that the latter one looks better, but once the collection size gets larger than your screen, it will not immediately be obvious what the problem is. 

So I guess version one!

**Comment by MarkusAmshove** [comment]
Alright.

The change will be shipped with [1.15](https://github.com/MarkusAmshove/Kluent/blob/master/CHANGELOG.md)
